ietf/doc/views_doc.py

```diff
@@ -315,18 +305,12 @@ def document_main(request, name, rev=None, document_html=False):
             diff_revisions=get_diff_revisions(request, name, doc if isinstance(doc,Document) else doc.doc)
             simple_diff_revisions = [t[1] for t in diff_revisions if t[0] == doc.name]
             simple_diff_revisions.reverse()
-            if rev and rev != doc.rev: 
-                # No DocHistory was found matching rev - snapshot will be false
-                # and doc will be a Document object, not a DocHistory
-                snapshot = True
-                doc = doc.fake_history_obj(rev)
-            else:
-                html = doc.html_body()
-                if request.COOKIES.get("pagedeps") == "inline":
-                    js = Path(finders.find("ietf/js/document_html.js")).read_text()
-                    css = Path(finders.find("ietf/css/document_html_inline.css")).read_text()
-                    if html:
-                        css += Path(finders.find("ietf/css/document_html_txt.css")).read_text()
+            html = doc.html_body()
+            if request.COOKIES.get("pagedeps") == "inline":
+                js = Path(finders.find("ietf/js/document_html.js")).read_text()
+                css = Path(finders.find("ietf/css/document_html_inline.css")).read_text()
+                if html:
+                    css += Path(finders.find("ietf/css/document_html_txt.css")).read_text()
```

Comment:
I wonder if these should be read into constants instead, to avoid the per-request overheads?

Reply:
I think we really need to avoid inlining them, especially the JS